lib/rules/id-length.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Rule that warns when identifier names are shorter or longer
+ *               than the values provided in configuration.
+ * @author Burak Yigit Kaya aka BYK
```

Comment:
@byk Basically, you own own whatever code you write. The CLA says you are licensing that code to ESLint so we can include it under our license terms. It's not restricting anyone's ability to use the code, it's just asserting that you created it AND that's what gives you the right to allow us to use it.